src/bot/index.ts

```diff
@@ -75,25 +84,30 @@ if (process.env.USE_JSON_CONFIG === `true`) {
 
 
 const _loadDefaultEnv = () => {
-  process.env.SERVER_ID='2234567890'
-  process.env.ACCOUNT='2234567890'
-  process.env.NETWORK = process.env.NETWORK || 'testnet'
-  process.env.API_USER='user'
-  process.env.API_PASS='password'
-  process.env.PORT='3000'
-  process.env.IP='0.0.0.0'
-  process.env.WEB3_TESTNET_PROVIDER='https://rinkeby.infura.io/v3/5ffc47f65c4042ce847ef66a3fa70d4c'
-  process.env.WEB3_MAINNET_PROVIDER='https://mainnet.infura.io/v3/5ffc47f65c4042ce847ef66a3fa70d4c'
+  const defaultConfig = {
+    SERVER_ID: '2234567890',
+    ACCOUNT: '2234567890',
+    NETWORK: 'testnet',
+    API_USER: 'user',
+    API_PASS: 'password',
```

Comment:
```suggestion
    API_PASS: '',
```
нужно убрать дефолтные пароли